package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@frontside/graphgen",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
I had to manually release 1.2.1 to get the feature to a client, so this should be 1.2.2 I wonder why this isn't getting a conflict?